docs/tutorial/persistence.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Persistence
+id: persistence
+---
+# Persistence
+
+## Why content slowly disappears from Swarm
+The storage capacity of the whole Swarm network equals the the sum of the capacity of all connected nodes. As nobody is restricted to upload content to Swarm, eventually the storage capacity of the Swarm network reaches its limits and nodes face the choice to either stop accepting new chunks or delete content which is there already for a while. 
```

Comment:
As nobody is restricted to upload content to Swarm **(until postage stamps accounting is implemented)**

Reply:
Thanks for the comment. 
I deliberately left postage stamps out of this because

1. postage stamps are not in the release and we should not clutter the docs with what is upcoming
2. even if there are postage stamps, the argument still holds: nobody is restricted (but it will cost you some money).